draft-ietf-teep-protocol.md

```diff
@@ -1779,6 +1779,23 @@ TEEP Broker
   Agents are protected against such downgrade attacks based on
   features offered by the manifest itself.
 
+Replay Protection
+: TEEP supports replay protection as follows.
+  The transport protocol under the TEEP protocol might provide replay
+  protection, but may be terminated in the TEEP Broker which is not trusted
+  by the TEEP and so the TEEP protocol does replay protection itself.
```

Comment:
```suggestion
  by the TEEP architecture and so the TEEP protocol does replay protection itself.
```

Reply:
actually should be "TEEP Agent" not "TEEP architecture"